srcpkgs/colord-gtk/template

```diff
@@ -1,26 +1,27 @@
 # Template file for 'colord-gtk'
 pkgname=colord-gtk
-version=0.2.0
-revision=2
+version=0.3.0
+revision=1
 build_style=meson
 build_helper=gir
-configure_args="-Dtests=false -Dvapi=$(vopt_if vala true false)
- -Ddocs=$(vopt_if docs true false) -Dman=$(vopt_if man true false)"
-hostmakedepends="pkg-config intltool glib-devel libxslt
+configure_args="-Dtests=false $(vopt_bool vala vapi) $(vopt_bool docs docs)
+ $(vopt_bool man man)"
+hostmakedepends="pkg-config gettext glib-devel libxslt
  $(vopt_if docs gtk-doc) $(vopt_if vala vala-devel)
  $(vopt_if man docbook-xsl-ns)"
-makedepends="colord-devel gtk+3-devel"
+makedepends="colord-devel gtk+3-devel gtk4-devel"
 short_desc="GTK support library for colord"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="https://github.com/hughsie/colord-gtk"
+changelog="https://raw.githubusercontent.com/hughsie/colord-gtk/main/NEWS"
 distfiles="${FREEDESKTOP_SITE}/colord/releases/${pkgname}-${version}.tar.xz"
-checksum=2a4cfae08bc69f000f40374934cd26f4ae86d286ce7de89f1622abc59644c717
+checksum=b9466656d66d9a6ffbc2dd04fa91c8f6af516bf9efaacb69744eec0f56f3c1d0
 
 build_options="vala docs man"
-build_options_default="vala"
-desc_option_docs="Generate documents"
-desc_option_man="Generate manpage"
+build_options_default="vala man"
+desc_option_docs="Generate documentation"
+desc_option_man="Generate man pages"
```

Comment:
docs should be at least moved to the -devel subpackage

Reply:
Makes sense. I enabled docs by default now and I wonder if the build option is necessary at all...
Should I rename it to `gtk_doc` to make it consistent with other packages?
`gtk_doc` is also currently missing a description in options.description btw